instrumentation/nocode/README.md

```diff
@@ -0,0 +1,30 @@
+# "nocode" instrumentation
+
```

Comment:
I think there should be a section here that clarifies this featureset as "experimental" (or as otel now spells it, "development").